client/components/split-button/index.jsx

```diff
@@ -102,6 +103,8 @@ class SplitButton extends PureComponent {
 		const isEmptyOnClick = this.props.onClick === noop;
 		const onClick = isEmptyOnClick ? undefined : this.handleMainClick;
 
+		const popoverContextRef = popoverContext ?? this.popoverContext;
```

Comment:
This is done so that we can place the tooltip to the dropdown instead to the main button